lms/product/factory.py

```diff
@@ -34,4 +35,13 @@ def _get_family(request):
     if "custom_canvas_course_id" in request.lti_params:
         return Product.Family.CANVAS
 
-    return Product.Family.UNKNOWN
+    # Finally try to match using the stored family_code in the application instance
```

Comment:
Ninja comment: I think we should point out this is actually only for the outcomes routes, and potentially consider limiting the times we apply this (like `if request.matched_route` above) to make it clear where this is for. We could expand it as necessary if we found we had to, which might for documentation of a sort of the places this is required.

Otherwise this would be very hard to prove backwards to only apply to that one place.